File: python_programming/notes/conditionals.py
# AE 7th Conditionals Notes

print("You: Can I go to my friends house? Please!")

homework_done = input("Did you do your homework?\n").strip().capitalize()

if homework_done == "Yes":
    print("Yes you can go, BUT you have to be home by 9.")
else: 
    print("Go do your homework!")

grade = 11

if grade >= 90:
    if grade > 100:
        print("You cheated that isn't possible")
    else:
        print(f"You have {grade}% which is an A!")
elif grade >= 80:
    print(f"You have {grade}% which is a B")
    print("")
else:
    print(f"You have a {grade}% which is a C or lower :(")

name = input("What is your name?\n")

if name == "Ms LaRose":
    print("You are the teacher,")
elif name == "Tia" or name == "Ashley":
    print("You are the TA")
else:
    print(f"Hello {name}, you are a student at UCAS")

File: notes/conditionals.c
// AE 7th Conditionals Notes
#include <stdio.h>
#include <string.h>
/* Logical Operators
python: and, or not
C: &&(and) ||(or) !(not)
*/

int main(void){
    int grade;
    char name[50];
    printf("Enter your grade percentage: ");
    scanf("%d", &grade);

    printf("Enter your name: ");
    scanf("%s", &name);

    //printf("&d\n", strcmp(name, "Ms. LaRose"));
    if(strcmp(name, "Ms. LaRose") == 0){
        printf("You don't get a grade because your a teacher :)\n");
    }else if(grade >= 90){
        printf("You have an A!\n");
    }else if(grade >= 80){
        printf("You have a B\n");
    }else if(grade >= 70){
        printf("You have a C\n");
    }else{
        printf("You are failing :(\n");
    }

    return 0;
}